pages/index.mdx

```diff
@@ -141,7 +144,9 @@ Alternatively, take a look at the following broader sections:
 
 ### Feeling a bit uncomfortable? [#next-3]
 
-If you ever get stuck, don't hesitate to reach out to Tact's flourishing community:
+If you ever get stuck, try searching — the search box is right at the top of the documentation. There is also a handy `Ctrl + K` shortcut to quickly focus and start the search as you type.
```

Comment:
maybe use `<kbd>` for the keyboard shortcut?

Reply:
Maybe, but it doesn't get underlined like inline code block does:

![image](https://github.com/user-attachments/assets/01622cb5-39d1-49cb-b444-18e806f7b47f)

Reply:
<img width="562" alt="image" src="https://github.com/user-attachments/assets/807b7cde-1fa4-44d9-80de-27b9d522656b">
github makes it prettier then :)

Reply:
this is the rendered version of this source: `<kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Space</kbd>`

Reply:
One downside there is that such notation gets split by lines because of the +